devops/scripts/install_drivers.sh

```diff
@@ -108,33 +108,38 @@ InstallIGFX () {
   echo "Compute Runtime version $CR_TAG"
   echo "CM compiler version $CM_TAG"
   echo "Level Zero version $L0_TAG"
-  IS_IGC_DEV=$(CheckIGCdevTag $IGCTAG)
-  IGNORE_CHECKSUM=false
-  DPKG_OPTIONS=""
-  if [ "$IS_IGC_DEV" == "Yes" ]; then
-    echo "IGC dev git hash $IGC_DEV_VER"
-    get_pre_release_igfx $IGC_DEV_URL $IGC_DEV_VER
-    IGNORE_CHECKSUM=true
-    DPKG_OPTIONS=" --force-depends-version"
-  else
-    echo "IGC version $IGC_TAG"
-    get_release intel/intel-graphics-compiler $IGC_TAG \
-      | grep ".*deb" \
-      | wget -qi -
-  fi
+  echo "IGC version $IGC_TAG"
+  get_release intel/intel-graphics-compiler $IGC_TAG \
+    | grep ".*deb" \
+    | wget -qi -
   get_release intel/compute-runtime $CR_TAG \
     | grep -E ".*((deb)|(sum))" \
     | wget -qi -
   # Perform the checksum conditionally and then get the release
-  ( [ "$IGNORE_CHECKSUM" ] || sha256sum -c *.sum ) && \
+  sha256sum -c *.sum  && \
   get_release intel/cm-compiler $CM_TAG \
     | grep ".*deb" \
     | grep -v "u18" \
     | wget -qi -
   get_release oneapi-src/level-zero $L0_TAG \
     | grep ".*deb" \
     | wget -qi -
-  dpkg -i $DPKG_OPTIONS *.deb && rm *.deb *.sum
+  dpkg -i *.deb && rm *.deb *.sum
+  IS_IGC_DEV=$(CheckIGCdevTag $IGCTAG)
+  if [ "$IS_IGC_DEV" == "Yes" ]; then
+    # Dev IGC deb package did not include libopencl-clang
+    # So we need to backup and install it from release igc.
```

Comment:
@bashbaug , do you know who can fix that on the IGC side?

Reply:
I would suggest starting with @pszymich.

Reply:
Thanks @aelovikov-intel and @bashbaug ! I had a quick look at https://github.com/intel/intel-graphics-compiler/blob/b077b43fa6193c97bcd2f3959f6d11e185d6fef3/.github/workflows/build-IGC.yml#L39C1-L41C56 , looks like IGC is also doing the hacky install/copy/package due to the fact that there is no libopencl-clang deb available.

I have contacted the libopencl-clang team to see whether we can build/release packaged libopencl-clang deb file in https://github.com/intel/opencl-clang/releases too. If that is fixed, then we should be able to remove this workaround, and also the workaround in IGC CI.